model-optimizer/mo/__main__.py

```diff
@@ -0,0 +1,13 @@
+import sys
```

Comment:
Why you didn't add the legal header for this file? I think that we must do this if the file structure allows us to do this which I believe is true.

Reply:
Fair point